src/Transports/MassTransit.RabbitMqTransport/RabbitMqTransport/RabbitMqBasicConsumer.cs

```diff
@@ -167,9 +160,9 @@ public void HandleModelShutdown(object model, ShutdownEventArgs reason)
                 var context = new RabbitMqReceiveContext(exchange, routingKey, _consumerTag, deliveryTag, bodyBytes, redelivered, properties,
                     _context, _receiveSettings, _model, _model.ConnectionContext);
 
-                var added = _pending.TryAdd(deliveryTag, context);
-                if (!added && deliveryTag != 1) // DIRECT REPLY-TO fixed value
-                    LogContext.Warning?.Log("Duplicate BasicDeliver: {DeliveryTag}", deliveryTag);
+                // var added = _pending.TryAdd(deliveryTag, context);
```

Comment:
Yep, this is dropped alright. This will flood the logs.

Reply:
Should we use Debug in `ConsumerAgentt` and Warning here?

Reply:
Probably need a virtual IsDuplicate method that returns true by default but RMQ can check for 1 and return false.

Reply:
agreed